docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -173,87 +234,4 @@ Here's each step:
-  - _NOTE: The certificate MUST match the domain you plan to use for your Metabase install._
-- Scroll to the bottom of the page and click `Save` in the lower right.
-  - _NOTE: Your Environment will begin updating with your new change. You will have to wait for this to complete before making additional updates._
-  - _IMPORTANT: Once this change is made you will no longer be able to access your Metabase instance at the `*.elasticbeanstalk.com` URL provided by Amazon because it will result in a certificate mismatch. To continue accessing your secure Metabase instance you must [Setup a DNS CNAME](#setup-dns-cname)._
-
-Once your application is working properly over HTTPS we recommend setting an additional property to force non-HTTPS clients to use the HTTPS endpoint.
-
-- Click on `Configuration` on the left hand sidebar.
-- Scroll down to `Software Configuration` under the _Web Tier_ section and click the gear icon to edit those settings.
-- Under `Environment Properties` add an entry for `NGINX_FORCE_SSL` with a value of `1`.
-- Scroll to the bottom of the page and click `Apply` in the lower right, then wait for your application to update.
-
-# Setting the JVM Timezone
-
-It's best to set your JVM timezone to match the timezone you'd like all your reports to come in. You can do this by adding the `JAVA_TIMEZONE` environment variable.
-
-- Click on `Configuration` on the left hand sidebar.
-- Scroll down to `Software Configuration` under the _Web Tier_ section and click the gear icon to edit those settings.
-- Under `Environment Properties` add the following.
-  - `JAVA_TIMEZONE` with a value such as `US/Pacific`.
-- Scroll to the bottom of the page and click `Apply` in the lower right, then wait for your application to update.
-
-# Using Papertrail for logging on AWS
-
-This provides a simple way to use the Papertrail logging service for collecting the logs for you Metabase instance in an easy to read location.
-
-- Click on `Configuration` on the left hand sidebar.
-- Scroll down to `Software Configuration` under the _Web Tier_ section and click the gear icon to edit those settings.
-- Under `Environment Properties` add the following entries:
-  - `PAPERTRAIL_HOST` - provided by Papertrail
-  - `PAPERTRAIL_PORT` - provided by Papertrail
-  - `PAPERTRAIL_HOSTNAME` - the name you want to see showing up in Papertrail for this server
-- Scroll to the bottom of the page and click `Apply` in the lower right, then wait for your application to update.
-
-_NOTE: Sometimes these settings will not apply until you restart your application server, which you can do by either choosing `Restart App Server(s)` from the Actions dropdown or by deploying the same version again._
+Once a new version is deployed you can safely delete the old Application Version if desired. We recommend keeping at least one previous version available for a while in case you desire to revert for any reason.
```

Comment:
```suggestion
Once a new version is deployed, you can safely delete the old Application Version. We recommend keeping at least one previous version available for a while in case you want to revert for any reason.
```